packages/client/src/v2-events/features/events/useEvents.ts

```diff
@@ -10,76 +10,138 @@
  */
 
 import { storage } from '@client/storage'
-import { api, utils } from '@client/v2-events/trpc'
+import { api, queryClient, utils } from '@client/v2-events/trpc'
 import { EventDocument } from '@events/schema'
-import { useQueryClient, useSuspenseQuery } from '@tanstack/react-query'
+import { useSuspenseQuery } from '@tanstack/react-query'
 
 export function preloadData() {
   utils.config.get.ensureData()
 }
 
+utils.event.actions.declare.setMutationDefaults(({ canonicalMutationFn }) => ({
```

Comment:
Do you have any issues when running this inline?

I receive something along the lines of "cannot use utils until it has been initialized"

Reply:
Noticed this as well but it got cleared away automatically. Something about import/export order and evaluation. Not yet 100% sure